src/store/utilities/export.js

```diff
@@ -15,19 +16,28 @@ function formatObject(obj) {
   return obj;
 }
 
+
+function mungeStories(stories, geometries) {
+  return stories.map((story) => {
+    const geometry = JSON.parse(JSON.stringify(
+      _.find(geometries, { id: story.geometry_id }),
+    ));
+    return {
+      ...story,
+      spaces: story.spaces,
+      geometry,
+      geometry_id: undefined,
```

Comment:
do we want undefined as the value instead of null?

Reply:
yes, we want `undefined` so that the key doesn't show up at all in the JSON.stringify'd output. Think `delete obj.geometry_id`, but then I would have needed another couple lines.